Change default shell to /bin/sh and disable mail spool creation

--- a/src/useradd.c
+++ b/src/useradd.c
@@ -91,7 +91,7 @@ static const char *def_groups = "";
 static const char *def_gname = "other";
 static const char *def_home = "/home";
-static const char *def_shell = "/bin/bash";
+static const char *def_shell = "/bin/sh";
 static const char *def_template = SKEL_DIR;
 static const char *def_usrtemplate = USRSKELDIR;
-static const char *def_create_mail_spool = "yes";
+static const char *def_create_mail_spool = "no";
 static const char *def_log_init = "yes";

 static long def_inactive = -1;
